src/titanium-data-table.ts

```diff
@@ -0,0 +1,347 @@
+import '../lib/titanium-svg-button';
+import '../lib/titanium-loading-indicator';
+
+import {css, customElement, html, LitElement, property, queryAll} from 'lit-element';
+import {TitaniumDataTableItem} from './titanium-data-table-item';
+
+@customElement('titanium-data-table')
+export class TitaniumDataTable extends LitElement {
+  @property() title: string;
+  @property() take: number;
+  @property() page: number = 0;
+  @property() count: number;
+  @property() items: Array<any>;
+  @property({type: Boolean, attribute: 'single-select', reflect: true})
+  singleSelect: boolean;
```

Comment:
If singleSelect is changed from the parent and multiple items are selected, deselectAll should probably be called

Reply:
Not worried about this, single select isn't made to be toggled during use.  I dont see a use case.  Let me know if you can think of a use case.